InventorySales/Model/IsProductSalableCondition/BackOrderNotifyCustomerCondition.php

```diff
@@ -73,15 +86,23 @@ public function execute(string $sku, int $stockId, float $requestedQty): Product
                 return $this->productSalableResultFactory->create(['errors' => []]);
             }
 
-            $backOrderQty = $requestedQty - $stockItemData[GetStockItemDataInterface::QUANTITY];
-            if ($backOrderQty > 0) {
+            $reservedQty = $this->getReservedQty(
```

Comment:
Please, take a look on `\Magento\InventorySalesApi\Api\GetProductSalableQtyInterface`, looks like you can replace introduced logic with one call.

Reply:
Resolved